src/js/utils/index.js

```diff
@@ -11,13 +11,13 @@ export * from './fil-d-ariane-utils';
 export * from './token-local-storage';
 export { default as typeRequest } from './mes-services-types-request';
 
-const grafanaBaseUri = conf.APP.GRAFANA_URI;
+const monitoringBaseUri = conf.APP.MONITORING_URI;
 
 const makeParamFromIdService = (id) =>
 	id
 		.split('/')
 		.filter((s) => s.trim().length > 0)
 		.join('_');
 
-export const getGrafanaServiceUrl = (service) =>
-	`${grafanaBaseUri}${makeParamFromIdService(service.id)}`;
+export const getMonitoringServiceUrl = (service) =>

```

Comment:
I don't think is this used anymore. A merged MR from few days ago introduced a new way to generate the monitoring url associated to a service which is now based on what the API is sending.  
So this is probably no longer used.